test/browser/clubs-page.test.jsx

```diff
@@ -298,6 +298,15 @@ describe("ClubsPage.ModalAddOrChangeYourClub", function() {
       should(modal.state.latitude).equal(null);
     });
 
+    it("requires user to read fact sheet", function() {
+      teachAPI.emit('username:change', 'foo');
```

Comment:
I see `username:change` being emitted across several places here - sorry never paid too much attention on the Teach API end before.  Does `teachAPI.emit('username:change', 'foo')` mean we are now logged in as user "foo"?